src/textual/renderables/sparkline.py

```diff
@@ -73,13 +74,15 @@ def __rich_console__(
         minimum, maximum = min(self.data), max(self.data)
         extent = maximum - minimum or 1
 
-        buckets = list(self._buckets(self.data, num_buckets=self.width))
+        buckets = tuple(self._buckets(self.data, num_buckets=width))
```

Comment:
Just nothing that along the way, and again only in files and functions that I touched, I changed `lists` to `tuples` if they are thought of as immutable; I changed `list` to `Sequence` in type hints when the same applies.